kblocks/extras/layers/scale.py

```diff
@@ -27,7 +27,7 @@ def get_config(self):
 
     def _scale_shape(self, input_shape):
         del self
-        return (input_shape[-1],)
+        return input_shape[-1],
```

Comment:
I much prefer `(x,)` for single-element tuples (and it's what `black` enforces). It's also why I tend to go for `(a, b)` where `a, b` is suffice - for consistency.

Reply:
Oh, in that case would you accept blackening your whole codebase?

Reply:
... it's supposed to be, but I might have forgotten to run it on files that haven't changed since I made that decision... if not I'll merge this PR then fix it up in a separate PR.